instrumentation/clickhouse/clickhouse-client/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/clickhouse/ClickHouseAttributesGetter.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.clickhouse;
+
+import io.opentelemetry.api.internal.StringUtils;
```

Comment:
this is an internal implementation class for opentelemetry-api, we generally try to avoid using such classes